lib/classes/PluginManager.js

```diff
@@ -23,7 +23,14 @@ const requireServicePlugin = (servicePath, pluginPath, localPluginsPath) => {
       if (!isModuleNotFoundError(error, absoluteLocalPluginPath)) throw error;
     }
   }
-  return require(cjsResolve(servicePath, pluginPath).realPath);
+  try {
+    return require(cjsResolve(servicePath, pluginPath).realPath);
+  } catch (error) {
+    if (!isModuleNotFoundError(error, pluginPath)) throw error;
+  }
+
+  // as a last ditch effort try location where serverless started from
+  return require(cjsResolve(path.dirname(require.main.filename), pluginPath).realPath);
```

Comment:
I think it's fine to simple use `__dirname` (as `require.main` does not necessarily may point expected path)